x-pack/legacy/plugins/siem/public/components/ml/anomalies_host_table.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiInMemoryTable, EuiPanel } from '@elastic/eui';
+import styled from 'styled-components';
+import { useAnomaliesTableData } from './use_anomalies_table_data';
+import { HeaderPanel } from '../header_panel';
+
+import * as i18n from './translations';
+import { getAnomaliesHostTableColumns } from './get_anomalies_host_table_columns';
+import { convertAnomaliesToHosts } from './convert_anomalies_to_hosts';
+import { BackgroundRefetch } from '../load_more_table';
+import { LoadingPanel } from '../loading';
+import { getIntervalFromAnomalies } from './get_interval_from_anomalies';
+import { getSizeFromAnomalies } from './get_size_from_anomalies';
+import { dateTimesAreEqual } from './date_time_equality';
+import { AnomaliesTableProps } from './types';
+
+const BasicTableContainer = styled.div`
+  position: relative;
+`;
+
+const sorting = {
+  sort: {
+    field: 'anomaly.severity',
+    direction: 'desc',
+  },
+};
+
+export const AnomaliesHostTable = React.memo<AnomaliesTableProps>(
+  ({ startDate, endDate, narrowDateRange }): JSX.Element => {
+    const [loading, tableData] = useAnomaliesTableData({
+      influencers: [],
+      startDate,
+      endDate,
+      threshold: 0,
+    });
+
+    const hosts = convertAnomaliesToHosts(tableData);
+    const interval = getIntervalFromAnomalies(tableData);
+    const columns = getAnomaliesHostTableColumns(startDate, endDate, interval, narrowDateRange);
+    const pagination = {
+      pageIndex: 0,
+      pageSize: 10,
+      totalItemCount: getSizeFromAnomalies(tableData),
+      pageSizeOptions: [5, 10, 20, 50],
+      hidePerPageOptions: false,
+    };
+    return (
+      <EuiPanel>
+        <BasicTableContainer>
+          {loading && (
+            <>
+              <BackgroundRefetch />
+              <LoadingPanel
+                height="100%"
+                width="100%"
+                text={`${i18n.LOADING} ${i18n.ANOMALIES}`}
+                position="absolute"
+                zIndex={3}
+                data-test-subj="anomalies-host-table-loading-panel"
+              />
+            </>
+          )}
```

Comment:
nit: No change necessary as part of this PR, but looks like this block might be ready to be its own component. We've got three usages of this explicitly, and four others that are similar but make use of `LoadingPanelContainer`.

Reply:
Let me swap over to `<LoadingOverlay />` as that seems to be more commonly used one and remove the export off of `<BackgroundRefetch />` for now. Then we can refactor that a bit more down the road.

Reply:
Actually if I switch to using that one it is slightly off margin wise compared to the `BackgroundRefetch` that the load more tables are using :-/

So I think I will leave this as is for now.